common/assets/scss/overrides/_govuk-frontend.scss

```diff
@@ -32,6 +32,27 @@
   padding: govuk-spacing(1) govuk-spacing(2);
 }
 
+// Add support to show when a field contains a message
+.govuk-form-group--message:not(.govuk-form-group--error) {
```

Comment:
Same question as the one below really...  Could it be done by adding an app-specific class instead